cishouseholds/pipeline/regex_patterns.py

```diff
@@ -32,11 +32,173 @@
+        "CHAPL[AI]*N|VICAR|CLERGY|MINISTER|PREACH|CHURCH|PARISH",  # religous services
+        "I[ \\.]*T\\.?\\b|DIGIT|WEBSITE|NETWORK|DEVELOPER|SOFTWARE|SYSTEM|CLERICAL",  # IT
+        "CHAIR|CHARITY|CITIZEN|CIVIL|VOLUNT|LIBRAR|TRANSLAT|INVESTIGAT|FIRE ?(WO)?(M[AE]N|FIGHT)|POLICE|POST *(WO)*MAN|PRISON|FIRST AID|SAFETY|TAX|GOVER?NMENT",  # noqa:E501 public service
+        "CONTRACTOR|COUNCIL|MEDICAL SCHOOL|^RETIRED",
+    ]
+)
+
+non_patient_facing_regex = "|".join(
+    [
+        "ONLINE|ZOOM|MICROSOFT|MS TEAMS|SKYPE|GOOGLE HANGOUTS?|REMOTE|VIRTUAL",
+        "(ONLY|OVER THE) (TELE)?PHONE|((TELE)?PHONE|VIDEO) (CONSULT|CALL|WORK|SUPPORT)",
+        "(NO[TN]( CURRENTLY)?|NEVER) (IN PERSON|FACE TO FACE)",
+        "SH[EI]+LDING|WORK(ING)? (FROM|AT) HOME|HOME ?BASED|DELIVER(Y|ING)? (PRE|PER)SCRI",
+        "(?<!NOT )OFFICE BASED",
+        "((?=.*?(111|119|999|911|NHS|TRIAGE|EMERGENCY))(?=.*?(ADVI[SC][OE]R|RESPONSE|OPERAT|CALL (HANDLER|CENT(RE|ER)|TAKE)",  # noqa:E501
+        "(TELE)?PHONE|TELE(PHONE)?|COVID))(^(?!.*(CUSTOMER SERVICE|SALES)).*))",  # call handler
+        "RECEPTION",
+        "S.?C+R+.?T+.?R+Y",
+        "P.?RS+.?N+.?L AS+IS+T+AN+",
+        "CLER(K|ICAL)",
+        "ADMIN|MANAGER",
+    ]
+)
+
+outpatient_exclusions = "|".join(["LOCAL COUNCIL", "DISCHARGE", "BUSINESS"])
+support_roles = "|".join(
+    [
+        "RECEPTION",
+        "ASSIST[AE]NT",
+        "S.?C+R+.?T+.?R+Y",
+        "PA",
+        "P.?RS+.?N+.?L AS+IS+T+AN+",
+        "ADMIN",
+        "CLER(K|ICAL)",
+        "SUP+ORT *WORKER",
+        "TRAINEE",
+        "JUNIOU?R",
+    ]
+)
+healthcare_positive_regex = "|".join(
+    [  # noqa: E501  # counsellor
+        f"((?=.*?({support_roles}))(?=.*?({healthcare_roles}))(^(?!.*({outpatient_exclusions})).*))",  # noqa: E501 # other location dependent workers
+        "((?=.*?(111|119|999|911|NHS|TRIAGE|EMERGENCY))(?=.*?(ADVI[SC][OE]R|RESPONSE|OPERAT|CALL (HANDLER|CENT(RE|ER)|TAKE)|(TELE)?PHONE|TELE(PHONE)?|COVID))(^(?!.*(CUSTOMER SERVICE|SALES)).*))",  # noqa: E501  # call handler
+        "|".join(["^" + x for x in healthcare_roles.split("|")]),
+    ]
+)
+
+healthcare_pattern = RegexPattern(
+    positive_regex_pattern=healthcare_positive_regex,
+    negative_regex_pattern=non_healthcare_regex + "|" + social_care_positive_regex,
+)
+
+socialcare_pattern = RegexPattern(
+    positive_regex_pattern=social_care_positive_regex,
+    negative_regex_pattern=healthcare_positive_regex,
+)
+
+patient_facing_pattern = RegexPattern(
+    positive_regex_pattern=healthcare_positive_regex + "|" + additional_healthcare_roles,
+    negative_regex_pattern=non_patient_facing_regex + "|" + non_healthcare_regex,
+)
+
+non_patient_facing_pattern = RegexPattern(positive_regex_pattern=non_patient_facing_regex, negative_regex_pattern=None)
+
 work_from_home_pattern = RegexPattern(
     positive_regex_pattern="(W(K|ORK.*?) F(ROM?) H(OME?))|(WFH)",
     negative_regex_pattern="HOMELESS",
 )
 
+healthcare_bin_pattern = RegexPattern(
```

Comment:
not sure where we use this either. This looks like it would produce similar output to health_care_positive_roles and not certain where it's being used?